noxfile.py

```diff
@@ -369,7 +369,7 @@ def compliance(session):
         session.skip("Compliance tests were not found")
 
     session.install("--pre", "grpcio")
-    session.install("--pre", "--no-deps", "--upgrade", "sqlalchemy<2.0.0")
+    session.install("--pre", "--no-deps", "--upgrade", "sqlalchemy>=1.4.16,<2.1")
```

Comment:
2.1 isn't out yet. Let's delete the max version, especially here in prerelease tests, so we get a heads up if we do end up being incompatible with it.